pilz_trajectory_generation/test/test_robots/prbt/test_data/testdata_sequence.xml

```diff
@@ -205,6 +205,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       <sequenceCmd name="open_gripper" type="gripper" blend_radius="0.0" />
       <sequenceCmd name="close_gripper" type="gripper" blend_radius="0.0" />
     </blend>
+    <blend name="SequenceBlend1">
+      <sequenceCmd name="Ptp1"            type="ptp_joint_cart"     blend_radius="0.1" />
+      <sequenceCmd name="lin2"            type="lin_cart"           blend_radius="0.0" />
+      <sequenceCmd name="lin3"            type="lin_cart"           blend_radius="0.0" />
+    </blend>
+    <blend name="SequenceBlend2">
+      <sequenceCmd name="Ptp1"            type="ptp_joint_cart"     blend_radius="0.0" />
+      <sequenceCmd name="lin2"            type="lin_cart"           blend_radius="0.1" />
+      <sequenceCmd name="lin3"            type="lin_cart"           blend_radius="0.0" />
+    </blend>
+    <blend name="SequenceBlend3">
+      <sequenceCmd name="Ptp1"            type="ptp_joint_cart"     blend_radius="0.0" />
+      <sequenceCmd name="lin2"            type="lin_cart"           blend_radius="0.0" />
+      <sequenceCmd name="lin3"            type="lin_cart"           blend_radius="0.1" />
+    </blend>
```

Comment:
I got rid of them altogether :+1:

Reply:
Perfect! But I still think we don't need the three cases.
* There is already a test case "lastBlendingRadiusNonZero"
* the other two can be combined into: "0.0 - 0.1 - 0.0" since only the transitions matter.